go.mod

```diff
@@ -1,20 +1,28 @@
-module k8s.io/kube-state-metrics
+module k8s.io/kube-state-metrics/v2
 
 require (
+	github.com/Azure/go-autorest/autorest v0.10.0 // indirect
```

Comment:
Curious of all the changes that were added?

Reply:
I just triggered `go mod tidy` but maybe I should have done it in a different way?

Reply:
It's not a problem at all, mainly asking as we will be porting this to release 2.0 branch and did not want to introduce any new not needed changes, but don't think its a problem here.